### ISSUE TITLE
Read-only rootfs is causing issues

### ISSUE THREAD

**Issue by stv0g** [bootc/netbox-chart#34]
```
2021/02/07 15:37:13 [warn] 1#1 Unit is running unprivileged, then it cannot use arbitrary user and group.
--
2021-02-07 16:37:14 | 2021/02/07 15:37:13 [alert] 1#1 Unable to create certificates storage directory: mkdir(/opt/unit/state/certs/) failed (30: Read-only file system)
2021-02-07 16:37:14 | 2021/02/07 15:37:13 [alert] 1#1 bind(7, unix:/opt/unit/unit.sock.tmp) failed (30: Read-only file system)
```

I would suggest changing the default security context:

```diff
securityContext:
-  readOnlyRootFilesystem: true
+  readOnlyRootFilesystem: false
```

**Comment by Jacq** [comment]
Updated both general and nginx readOnlyRootFilesystem to false, but still the Read-only file system error is showing on boot.

**Comment by stv0g** [comment]
I also had to disable the non-root securityContext

**Comment by Jacq** [comment]
No luck with that also, I will probably will clean everything and start the containers again.

**Comment by bootc** [comment]
This will be fixed with version 3.0.0 of the chart, which I'm currently working on in `master`. The fix for this particular issue is in https://github.com/bootc/netbox-chart/commit/38f4b943abdc353d1e8d13a83c98c1b03c2601db and is due to the switch from Gunicorn to NGINX Unit.
